godtools/App/Features/Onboarding/OnboardingQuickStartView.xib

```diff
@@ -0,0 +1,103 @@
+                    <subviews>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Quick Start" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="NBq-0r-pAH" userLabel="Title label">
+                            <rect key="frame" x="30" y="40" width="354" height="31.5"/>
+                            <fontDescription key="fontDescription" name="SFProDisplay-Light" family="SF Pro Display" pointSize="26"/>
+                            <color key="textColor" red="0.23137254900000001" green="0.64313725489999995" blue="0.85882352939999995" alpha="1" colorSpace="custom" customColorSpace="calibratedRGB"/>
+                            <nil key="highlightedColor"/>
+                        </label>
+                        <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="-1" estimatedSectionHeaderHeight="-1" sectionFooterHeight="-1" estimatedSectionFooterHeight="-1" translatesAutoresizingMaskIntoConstraints="NO" id="l4Q-zP-eGG">
+                            <rect key="frame" x="30" y="91.5" width="354" height="584.5"/>
+                            <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        </tableView>
+                    </subviews>
+                    <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                    <constraints>
+                        <constraint firstItem="NBq-0r-pAH" firstAttribute="top" secondItem="WFX-ff-YxE" secondAttribute="top" constant="40" id="84I-lT-bib"/>
+                        <constraint firstAttribute="trailing" secondItem="l4Q-zP-eGG" secondAttribute="trailing" constant="30" id="Ca8-FY-dOM"/>
+                        <constraint firstItem="NBq-0r-pAH" firstAttribute="leading" secondItem="WFX-ff-YxE" secondAttribute="leading" constant="30" id="PU4-xU-xiE"/>
+                        <constraint firstAttribute="trailing" secondItem="NBq-0r-pAH" secondAttribute="trailing" constant="30" id="TB2-lM-0zY"/>
+                        <constraint firstAttribute="bottom" secondItem="l4Q-zP-eGG" secondAttribute="bottom" constant="24" id="dAa-O5-HTa"/>
+                        <constraint firstItem="l4Q-zP-eGG" firstAttribute="top" secondItem="NBq-0r-pAH" secondAttribute="bottom" constant="20" id="kHf-Hq-wIh"/>
+                        <constraint firstItem="l4Q-zP-eGG" firstAttribute="leading" secondItem="WFX-ff-YxE" secondAttribute="leading" constant="30" id="vFo-B7-pe3"/>
+                    </constraints>
+                </view>
+                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Pke-Cm-Fv3" userLabel="Footer View">
```

Comment:
Hey @reldredge71 I'm thinking we need to add something here for shorter devices where not all of the quick actions will be visible.  Can you add a white fade those items would fade into, I think that will help see that there are more items.


![iPhone_5s](https://user-images.githubusercontent.com/59846460/142882866-dbd95516-7ee6-4aea-8624-c08fc2bb5dc1.png)

 
Something like this.



![fade](https://user-images.githubusercontent.com/59846460/142882879-27208948-4a11-4185-b89b-b2fa78486d88.png)

Reply:
I've been trying to get a gradient to work on this.  I wrapped the UITableView in a UIView and applied the gradient to that.  the "locations" of the points of the gradient don't make sense to me.   If the screen is so small that the table view starts scrolling, then I can't see the bottom fade-out at all.  It almost seems to me like the gradient's frame is taking its dimensions from the full height of the table view.... but the whole point of wrapping it in a UIView was to avoid that and have the gradient only fill the visible part of the table.  IDK what's up, you can play around with it if you have any ideas what the issue might be.

Reply:
Hey @reldredge71 that gradient does fill the entire view, I think in your case you'd have to set those locations with a combination of using clear color to get the desired effect.  Wrapping the table in a gradient view makes it more complex to calculate across device sizes too.
How I've done something similar in the past is have the tableview fill the entire screen and then setting the bottom contentInset of the tableview to match the height of the gradient area which overlaid the bottom of the tableview, that gave the effect of those items scrolling behind something. 
https://developer.apple.com/documentation/uikit/uiscrollview/1619406-contentinset